CMakeLists.txt

```diff
@@ -231,6 +231,16 @@ if (WIN32)
             string(REPLACE "/INCREMENTAL" "/INCREMENTAL:NO" ${LinkerFlag} ${${LinkerFlag}})
         endforeach()
     endif()
+
+    if (MSVC)
+        option(FILAMENT_SHORTEN_MSVC_COMPILATION "Shorten compile-time in Visual Studio" ON)
```

Comment:
I think we need to do one of two things: turn this `OFF` by default, or turn it `OFF` for CI builds (see [build-github.bat](https://github.com/google/filament/blob/main/build/windows/build-github.bat) and the `FILAMENT_WINDOWS_CI_BUILD` flag).

IIRC, we used to have `/MP` turned on for CI builds, but it led to instability due to limited memory on CI machines.

Reply:
Can we a a set of flags for CI  and one for local builds. This way, when/if we get better machines for CI et can easily add the option?